app/test_main.py

```diff
@@ -1,3 +1,20 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected_ages", [
+        (15, 15, [1, 1]),
+        (15, 40, [1, 5]),
+        (25, 25, [2, 2]),
+        (0, -1, [0, 0])
```

Comment:
The test case `(0, -1, [0, 0])` includes a negative age. Ensure that `get_human_age` is designed to handle negative ages appropriately. If negative ages are invalid, consider adding a test to check that an exception is raised.